tools/RepositoryManagement/DevHome.RepositoryManagement/Views/RepositoryManagementMainPageView.xaml

```diff
@@ -170,31 +194,85 @@
                                 ToolTipService.ToolTip="WinGet Configuration file found. Run it from the '...' menu to set up this repo"/>
                             <TextBlock
                                 Grid.Column="2"
-                                Text="{x:Bind ClonePath, Mode=OneWay}"
-                                ToolTipService.ToolTip="{x:Bind ClonePath, Mode=OneWay}" />
-                            <!-- TODO: Provide commit URL, name, author name, and time -->
+                                Text="{x:Bind ClonePath, Mode=OneWay}"/>
                             <TextBlock
+                                x:Uid="CommitNotFound"
+                                x:Name="CommitNotFoundTextBlock"
+                                Grid.Column="3"
+                                Visibility="{x:Bind HasCommitInformation, Converter={StaticResource TrueToCollapsedConverter}}">
+                                <ToolTipService.ToolTip>
+                                    <TextBlock x:Uid="CommitNotFound"/>
+                                </ToolTipService.ToolTip>
+                            </TextBlock>
+                            <StackPanel
                                 Grid.Column="3"
-                                Text="{x:Bind LatestCommit}"
-                                ToolTipService.ToolTip="{x:Bind LatestCommit}" />
+                                x:Name="CommitFoundStackPanel"
+                                Visibility="{x:Bind HasCommitInformation, Converter={StaticResource TrueToVisibleConverter}}"
+                                Orientation="Horizontal">
+                                <TextBlock Text="{x:Bind LatestCommitSHA}"/>
+                                <FontIcon
+                                    FontFamily="{StaticResource SymbolThemeFontFamily}"
+                                    Glyph="&#xEA81;"
+                                    FontSize="8"
+                                    VerticalAlignment="Top"
+                                    Margin="0, 5, 5, 0"/>
+                                <TextBlock Text="{x:Bind LatestCommitAuthor}"/>
+                                <FontIcon
+                                    FontFamily="{StaticResource SymbolThemeFontFamily}"
+                                    Glyph="&#xEA81;"
+                                    FontSize="8"
+                                    VerticalAlignment="Top"
+                                    Margin="0, 5, 5, 0"/>
+                                <TextBlock Text="{x:Bind MinutesSinceLatestCommit}"/>
```

Comment:
This doesn't seem like a helpful raw value to display. See my earlier comment changing this property to `DateTime LatestCommitTime`.

Then run this through a converter to customize how you'll display it. If you want to display how long ago it was, then compare that value to `DateTime.Now` and format it better than raw minutes. You'll also need to localize `"{0} days/hours/minutes ago"` rather than a raw incomprehensible integer.

Reply:
Very true.  I don't think Design accounted for this.  I also didn't ask about this issue during design.  I'm going to leave this here and talk to design on Monday.